src/Language/PureScript/Make/Actions.hs

```diff
@@ -97,7 +97,7 @@ data MakeActions m = MakeActions
   , readExterns :: ModuleName -> m (FilePath, Maybe ExternsFile)
   -- ^ Read the externs file for a module as a string and also return the actual
   -- path for the file.
-  , codegen :: CF.Module CF.Ann -> Docs.Module -> ExternsFile -> SupplyT m ()
+  , codegen :: CF.Module CF.Ann -> Docs.Module -> Maybe ExternsFile -> SupplyT m ()
```

Comment:
When performing `codegen` via `purs codegen` - we can create a stub `ExternsFile` via the `CoreFn.Module` - but it isn't actually the `ExternsFile` we want to write. I modified this so that we don't write a bogus `ExternsFile` during `purs codegen`

Reply:
Good idea - I can go ahead and do that now